Tests/DatadogTests/Datadog/Core/PerformancePresetTests.swift

```diff
@@ -137,4 +137,11 @@ class PerformancePresetTests: XCTestCase {
             )
         }
     }
+
+    func testPresetsUpdate() {
+        let preset = PerformancePreset(batchSize: .mockRandom(), uploadFrequency: .mockRandom(), bundleType: .mockRandom())
+        let updatedPreset = preset.updated(with: PerformancePresetOverride(maxFileSize: 0, maxObjectSize: 0))
```

Comment:
nit/ If we use `.mockRandom(min:max:)` (or `.mockRandom(min:max:otherThan:)` instead of hardcoding `0`-es this test would become much better. Instead of testing single value, it would be covering all possible configurations.